.gitlab/new-e2e_testing/ubuntu.yml

```diff
@@ -15,6 +15,26 @@
     TEAM: agent-platform
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --major-version $AGENT_MAJOR_VERSION --arch $E2E_ARCH --flavor $FLAVOR
 
+.new-e2e_script_upgrade:
+  variables:
+    TARGETS: ./tests/agent-platform/upgrade
+    TEAM: agent-platform
+    EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --arch $E2E_ARCH --flavor $FLAVOR
+  parallel:
```

Comment:
Do you think we could split matrix to be able to define two kinds of upgrade job:
- the ones that update from * version to 6 
- the ones that update from * version to 7
So we can create upgrade jobs that do not need to depend on both agent 6 and agent 7 deploy